uiowa/uiowa
# Classrooms - Fix classrooms view sort

Resolves https://github.com/uiowa/uiowa/issues/6767

# To Test

```
ddev blt ds --site=classrooms.uiowa.edu && ddev drush @classrooms.local uli spaces/find-university-classroom
```

- Update hook should fire, with no issues.
- Though there is a combined id field, it is not visible in the form or any of the displays.
- See that ABW rooms are showing up first before AJB.
- See that AJB W240 is the last AJB building
- Go to page 3 unfiltered and see LC N105 is right after LC N104
- Filter by Lindquist Center and see that LC N105 is right after LC N104
- See that the view A-Z sort options are the same as PROD and work as expected based on the card titles.
